extensions/uploadthing/src/lib/utils.tsx

```diff
@@ -122,3 +123,24 @@ export const getSignedUrls = async (files: { key: string }[]) => {
     }),
   );
 };
+
+export const getStatusIcon = (
+  status: "Deletion Pending" | "Failed" | "Uploaded" | "Uploading",
```

Comment:
nit: i would make this list a constant, and reuse it for mapping options in the dropdown

keeps things more consistent, and if we were to add a status, only need to update in one place